react/react1/week2/src/App.js

```diff
@@ -0,0 +1,61 @@
+import React, {useState, useRef, useEffect} from "react";
+import ToDoList from "./ToDoList";
+import { v4 as uuidv4 } from 'uuid';
+import Counter from "./Counter.js"
+
+
+const storageKey = "todoApp.todos" 
+
+function App() {
+  const [todos, setTodos] = useState([])
+  const todoNameRef = useRef()
```

Comment:
Generally, you shouldn't rely too much on useRef, especially in cases like this where you can still use useState().
So, I would suggest you define a new newTodoName state, which can be update by the onChange event of your input box like this:
```
const handleChange = (event) => {
    setNewTodoName(event.target.value);
  };
...
<input onChange={handleChange} type= "text"/>
```